del-slb-nncp.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: nmstate.io/v1                                                       
+kind: NodeNetworkConfigurationPolicy                                            
+metadata:                                                                       
+  name: slb
+spec:
+  capture:
+    primary-nic: interfaces.description == "primary"
+    secondary-nic: interfaces.description == "secondary"
+  desiredState:
+    interfaces:
+    - name: brcnv-iface
+      type: ovs-interface
+      state: absent
+    - name: brcnv
+      type: ovs-bridge
+      state: absent
+    - name: "{{ capture.primary-nic.interfaces.0.name }}"
+      type: ethernet
+      state: up
+      ipv4:
+        enabled: true
+        dhcp: true
+      ipv6:
+        enabled: true
+        dhcp: true
```

Comment:
Did we use IPv6 in the original script? If not, I think we should remove this. We are not guaranteed to have a DHCPv6 available AFAIK

Reply:
I would stick to what we had (IPv4 only) until they bring up a need for IPv6. The thing is, even if they will need IPv6, DHCPv6 is not the only way of assigning addresses, I believe it is the less common dynamic addressing mechanism in IPv6.